ui/vendor/hawk/browser.js

```diff
@@ -0,0 +1,793 @@
+'use strict';
+
+/*
+    HTTP Hawk Authentication Scheme
```

Comment:
There's no version number here - would be good to add one.
I'm guessing this might have also come from master?

Perhaps we should take from the latest tagged revision, and add a version number somewhere (either in the filename or else in this file comment)?
https://github.com/hueniverse/hawk/releases